libraries/chain/database.cpp

```diff
@@ -352,6 +352,22 @@ namespace golos { namespace chain {
             return _clear_votes_block > head_block_num();
         }
 
+        void database::set_store_account_metadata(store_metadata_modes store_account_metadata) {
+            _store_account_metadata = store_account_metadata;
+        }
+
+        void database::set_accounts_to_store_metadata(const std::vector<std::string>& accounts_to_store_metadata) {
+            _accounts_to_store_metadata = accounts_to_store_metadata;
+        }
+
+        bool database::store_metadata_for_account(const std::string& name) const {
+            if (_store_account_metadata != store_for_listed) {
+                return _store_account_metadata == store_for_all ? true : false;
```

Comment:
`return _store_account_metadata == store_for_all ? true : false;` is to complex.
Change it to the same, but more simple:
`return _store_account_metadata == store_for_all;`